meteor/client/ui/Settings/components/rundownLayouts/ShelfLayoutSettings.tsx

```diff
@@ -71,6 +71,21 @@ export default withTranslation()(
 							></EditAttribute>
 						</label>
 					</div>
+					<div className="mod mvs mhs">
+						<label className="field">
+							{t('Hide default AdLib Start/Execute options')}
+							<EditAttribute
```

Comment:
It seems wrong that `<EditAttribute />` is inside of the label. 
Usally labels are used like so:
```
<label for="username">Username<label/>
<input id=username />
<label for"password">Password<label/>
<input id="password"/>
```
I can see that you used the same approach as for the rest of this file, so I don't think you need to change it.
Just wanted to point it out.